nixops/util.py

```diff
@@ -383,3 +384,5 @@ def xml_expr_to_python(node):
 def parse_nixos_version(s):
     """Split a NixOS version string into a list of components."""
     return s.split(".")
+
+root_dir = os.path.dirname(os.path.dirname(__file__))
```

Comment:
`__file__` is not portable to some implementations of Python

Reply:
@coretemp tnx for suggestion

based on https://stackoverflow.com/a/20510502/3574379 `__file__` is not working in Python Shell, PyInstaller and Py2exe

do you propose to change `__file__` to `os.path.realpath('__file__')` everywhere?

Reply:
In my programs I pass the installation directory such that no `__file__` symbol is required.

Reply:
@coretemp not sure I understand how to pass `installation directory`. What if method that use the installation directory is used in tests, where there is no installation directory?

this trick doesnt work

```
root_dir = os.path.dirname(os.path.dirname(__file__))
print(root_dir, file=sys.__stdout__)
print(sys.executable, file=sys.__stdout__)
```

outputs 

```
[nix-shell:~/projects/nixops]$ python2 tests.py --collect-only -v
/home/srghma/projects/nixops
/nix/store/6b6lzbq74g95j8qlbrdjzkn2ghh81mgy-python-2.7.15/bin/python2
```